samples/matter/thermostat/README.rst

```diff
@@ -0,0 +1,280 @@
+
+  .. list-table:: Thermostat sample onboarding information
+     :header-rows: 1
+
+     * - QR Code
+       - QR Code Payload
+       - Manual pairing code
+     * - Scan the following QR code with the app for your ecosystem:
+
+         .. figure:: ../../../doc/nrf/images/matter_qr_code_thermostat.png
+            :width: 200px
+            :alt: QR code for commissioning the thermostat device
+
+       - MT:O4CT3KQS02ZZ 
+       - 548G00
+
+Configuration
+*************
+
+|config|
+
+Matter thermostat build types
+===========================
+
+The sample uses different configuration files depending on the supported features.
+Configuration files are provided for different build types and they are located in the application root directory.
+
+The :file:`prj.conf` file represents a ``debug`` build type.
+Other build types are covered by dedicated files with the build type added as a suffix to the ``prj`` part, as per the following list.
+For example, the ``release`` build type file name is :file:`prj_release.conf`.
+If a board has other configuration files, for example associated with partition layout or child image configuration, these follow the same pattern.
+
+.. include:: /config_and_build/modifying.rst
+   :start-after: build_types_overview_start
+   :end-before: build_types_overview_end
+
+Before you start testing the application, you can select one of the build types supported by the sample.
+This sample supports the following build types, depending on the selected board:
+
+* ``debug`` -- Debug version of the application - can be used to enable additional features for verifying the application behavior, such as logs or command-line shell.
+* ``release`` -- Release version of the application - can be used to enable only the necessary application functionalities to optimize its performance.
+* ``no_dfu`` -- Debug version of the application without Device Firmware Upgrade feature support - can be used for the nRF52840 DK, nRF5340 DK, nRF7002 DK, and nRF21540 DK.
+
+.. note::
+    `Selecting a build type`_ is optional.
+    The ``debug`` build type is used by default if no build type is explicitly selected.
+
+Device Firmware Upgrade support
+===============================
+
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_build_with_dfu_start
+    :end-before: matter_door_lock_sample_build_with_dfu_end
+
+FEM support
+===========
+
+.. include:: /includes/sample_fem_support.txt
+
+User interface
+**************
+
+.. include:: ../lock/README.rst
+    :start-after: matter_door_lock_sample_led1_start
+    :end-before: matter_door_lock_sample_led1_end
+
+Button 1:
+     If pressed for six seconds, it initiates the factory reset of the device.
+     Releasing the button within the six-second window cancels the factory reset procedure.
+
+Button 2:
+     Prints thermostat data to the console.
```

Comment:
What about nRF7002? There it also triggers the BLE advertisement when pressed longer.